example/src/pages/OTPInput.tsx

```diff
@@ -30,14 +31,19 @@ const OTPWrapper = ({ secret = false, validation = false }: WrapperProps) => {
   const onValidate = (v: string) => !validation || v === OTP_COMPARE;
 
   return (
-    <OTPInput
-      value={value}
-      onValueChange={onValueChange}
-      length={OTP_LENGTH}
-      secret={secret}
-      onValidate={onValidate}
-      errorMessage={"Wrong OTP"}
-    />
+    <>
+      <OTPInput
+        value={value}
```

Comment:
What about exposing also a view ref? As for a11y this'd let us focus on the element if we want. What do you think? The alternative is to wrap the componenti in an accessible `View` with a ref.

```ts
 <View accessible ref={pinPadViewRef}>
  <OTPInput
    secret
    value={pin}
    accessibilityLabel={I18n.t(
      "authentication.cie.pin.accessibility.label"
    )}
    onValueChange={setPin}
    length={CIE_PIN_LENGTH}
  />
</View>
```

Reply:
Added in [1a92ac5](https://github.com/pagopa/io-app-design-system/pull/248/commits/1a92ac5c175733a233550484d3ad081e534f409f)